operator/pkg/central/extensions/common_test.go

```diff
@@ -30,10 +29,11 @@ type secretReconciliationTestCase struct {
 	Other                  []ctrlClient.Object
 	InterceptedK8sAPICalls interceptor.Funcs
 
-	ExpectedCreatedSecrets     map[string]secretVerifyFunc
-	ExpectedError              string
-	ExpectedNotExistingSecrets []string
-	VerifyStatus               statusVerifyFunc
+	ExpectedCreatedSecrets       map[string]secretVerifyFunc
+	ExpectedSecretsAfterDeletion []string
```

Comment:
Initially, the test checked that secrets created by the operator are removed when the CR is deleted. This new fields specifies the secrets that are created but that should not be removed.